### ISSUE TITLE
Some help with hls/rtmp delays ? 

### ISSUE THREAD

**Issue by mrfrieza** [arut/nginx-rtmp-module#1226]
Hello,

Need your advice about two things : 

HLS : Set up the fragments at 6s, chunk size 8192 and works fine, but can't decrease the fragments time or it'll send me back a 206 error. 

RTMP : Delay at around 2 sec, but not as smooth as HLS and I'm kinda worried about giving up the RTMP link as it could be used from anyone to send a stream... rtmp://ip/app/streamkey, they'd just need to make their own streamkey and I wouldn't be able to do anything.. 

-------

Additional note : 

I've been using nginx on two PC, one is getting the stream and rendering the stream to twitch with rtmp AND send the stream to the other nginx that produces the HLS/DASH streams (so that I can keep my ip private to send just basic rtmp to twitch, and the other one (aws amazon ec2) produces the hls/dash streams that can be shared. 

So basically, I'd just need advices on reducing the delay, don't hesitate to send me your config file if you succeeded at reducing it !

Thanks !
